source/Ship.cpp

```diff
@@ -2075,7 +2076,8 @@ double Ship::TurnRate() const
 
 double Ship::Acceleration() const
 {
-	return attributes.Get("thrust") / Mass();
+	return (attributes.Get("thrust") ? attributes.Get("thrust")
+			: attributes.Get("afterburner thrust")) / Mass();
```

Comment:
Just because this function gets called a *lot* and attribute lookups by name are time-consuming, it would make sense here and in MaxVelocity() to store the thrust value in a local variable, i.e.:

```c++
double thrust = attributes.Get("thrust");
return (thrust ? thrust : attributes.Get("afterburner thrust")) / Mass();
```

Reply:
Added